### PR TITLE
185454321 client service pane

### PR DESCRIPTION
**Pull request by ZachCMP**
https://www.pivotaltracker.com/story/show/185454321

Relies on frontend PR: https://github.com/greenriver/hmis-frontend/pull/337

### CODE REVIEW

File: drivers/hmis/spec/requests/hmis/service_filter_spec.rb [hunk 1]
```diff
@@ -31,8 +31,8 @@
 
   let(:query) do
     <<~GRAPHQL
-      query EnrollmentServices($id: ID!, $filters: ServiceFilterOptions!) {
-        enrollment(id: $id) {
```
Comment (gigxz) [inline drivers/hmis/spec/requests/hmis/service_filter_spec.rb:35]: Do we have coverage for this elsewhere? I don't think we want to remove tests for resolving services on enrollment..

**Review by gigxz — COMMENTED** [review]
LGTM, things are working as expected. Fine to merge as soon as we're not losing test coverage (if we are)

**Comment by ZachCMP** [comment]
@gigxz Fixed. I just added tests for resolving services on both enrollment and client

**Comment by ZachCMP** [comment]
@gigxz Tests are passing now

**Pull request merged**